sesman/chansrv/chansrv_config.h

```diff
@@ -39,6 +39,27 @@ struct config_chansrv
 
     /** Whether to use nautilus3-compatible file lists for the clipboard */
     int use_nautilus3_flist_format;
+
+    /** RestrictInboundClipboard setting from sesman.ini */
+    int restrict_inbound_clipboard;
+
+    /** RestrictInboundClipboard=text setting from sesman.ini */
+    int restrict_inbound_clipboard_text;
```

Comment:
Is using bitmask here a good idea?

```
restrict_inbound_clipboard = TEXT | FILE;
```

Reply:
Then we can create a new function like `g_text2bool()` to convert from string `text, file, image` to bitmask expression and vise versa.  We can also create unit tests for the conversion.

Reply:
Agreed, and covered above.

Reply:
Thanks, I've changed to use bitmask.